components/src/components/AvatarGroup/AvatarGroup.tsx

```diff
@@ -33,7 +33,7 @@ export const AvatarGroup = ({
 }: Props) => {
   const membersCount = members.length
   const visibleMembers = members.slice(0, limit)
-  const variantSize = size < 6 ? 'small' : 'large'
+  const variantSize = +size < 6 ? 'small' : 'large'
```

Comment:
I actually found a way around this. you can just make the size prop type `'5' | '6'`, and turn this line into `size === '5' ? 'small' : 'large'`. we only use those 2 values in web lol

Reply:
us in 5 months: "wtf, why can I use '5' or '6' for size on AvatarGroup but not any other number???"

Reply:
these are the 2 perfect numbers